src/order/filter/at_top_bot.lean

```diff
@@ -3,7 +3,9 @@ Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Jeremy Avigad, Yury Kudryashov, Patrick Massot
 -/
+import tactic.linarith
 import order.filter.bases
+import algebra.group_with_zero.power
```

Comment:
I don't think we want to mix `order` and `algebra` here. What is the reason for this move?

Reply:
It thought it was strange to put in `analysis/normed_space/ordered.lean` some lemmas which don't involve a norm. I chose this file because it seemed related to `filter.tendsto_pow_at_top`, which is proven here. Where would you like them to be ?

Reply:
It could go to a new file `analysis/specific_asymptotics`. Maybe we should even create a new subdirectoy named `analysis/asymptotics` and put in it the files `asymptotics.lean`, `asymptotic_equivalent.lean`, `specific_asymptotics.lean` and possibly `specific_limits.lean`.